### PR TITLE
Adapt Federated

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+authors = ["NuCypher"]
+license = "GPL-3.0"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+
+#
+# NuCypher
+#
+umbral = "==0.1.2a2"
```
Comment (cygnusv) [inline pyproject.toml]: We should be using `==0.1.3a0`. I'm afraid this file may be inconsistent with current Pipfile on master.

Comment (cygnusv) [reply pyproject.toml]: Perhaps we should remove poetry config for the moment. Later on we can include these files in a dedicated PR, including changes to integrate it with CI

Comment (KPrasch) [reply pyproject.toml]: :checkered_flag: PR #622

Comment (cygnusv) [reply pyproject.toml]: You're fast, man!

**Pull request merged**